src/main/webapp/index.html

```diff
@@ -93,19 +91,19 @@ <h2>Links to Resources</h2>
       <h4>Learn to code</h4>
       <ul class="resource-link">
         <li>
-          <a href="https://bit.ly/2BNeFLw" target="_blank">CodeAcademy: teaches you how to code</a>
+          <a href="https://bit.ly/2BNeFLw" target="_blank">CodeAcademy: Teaches you how to code</a>
         </li>
         <li>
-          <a href="https://bit.ly/389Z6cO" target="_blank">Udemy: a gallery of online clases for all subjects</a>
+          <a href="https://bit.ly/389Z6cO" target="_blank">Udemy: A gallery of online clases for all subjects</a>
         </li>
         <li>
           <a href="https://bit.ly/2Bb4u3H" target="_blank">Java Tutorial for Complete Beginners</a>
         </li>
         <li>
-          <a href="https://bit.ly/3i4nZv8" target="_blank">geeksforgeeks: website that teaches you programer with examples and videos</a>
+          <a href="https://bit.ly/3i4nZv8" target="_blank">GeeksforGeeks: Website that teaches you programer with examples and videos</a>
         </li>
         <li>
-          <a href="https://bit.ly/2Ad8Wyj" target="_blank">Youtube: how start codeing videos</a>
+          <a href="https://bit.ly/2Ad8Wyj" target="_blank">YouTube:Videos about how start coding videos</a>
```

Comment:
This sentence uses videos twice, so probably get rid of the last "videos"
YouTube: Videos about how *to* start coding